include/seqan3/search/fm_index/detail/csa_alphabet_strategy.hpp

```diff
@@ -162,6 +162,30 @@ namespace sdsl
             m_C.load(in);
             read_member(m_sigma, in);
         }
+
+        template <typename archive_t>
+        void CEREAL_SAVE_FUNCTION_NAME(archive_t & ar) const
+        {
+            ar(CEREAL_NVP(m_C));
+            ar(CEREAL_NVP(m_sigma));
+        }
+
+        template <typename archive_t>
+        void CEREAL_LOAD_FUNCTION_NAME(archive_t & ar)
+        {
+            ar(CEREAL_NVP(m_C));
+            ar(CEREAL_NVP(m_sigma));
+        }
+
+        bool operator==(plain_byte_alphabet const & other) const noexcept
+        {
+            return (m_C == other.m_C) && (m_sigma == other.m_sigma);
+        }
+
+        bool operator!=(plain_byte_alphabet const & other) const noexcept
+        {
+            return !(*this == other);
+        }
         //!\endcond
     };
 
```

Comment:
(Why) is this file still necessary?

Reply:
In general?
The SDSL alphabet strategies map the occurring symbols to ranks. This reduces the memory consumption if not every symbol occurs, but increases the runtime (additional lookup in the map).
This file uses identity (maps to the rank), hence faster.

Though I'd argue this could probably go into the SDSL.

Reply:
> Though I'd argue this could probably go into the SDSL.

That's what I meant. Can you open a card for this?